### ISSUE TITLE
bin links seem to cause issues when installing via npm

### ISSUE THREAD

**Issue by danie1kr** [kensnyder/poly-crypto#14]
Installing poly-crypto via npm in the standard way results in errors during chmod for the cli files:
```bash
npm install poly-crypto
npm ERR! code ENOENT
npm ERR! syscall chmod
npm ERR! path /home/ubuntu/myapp/node_modules/poly-crypto/node ./cli/aes-pass-encrypt.js
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, chmod '/home/ubuntu/myapp/node_modules/poly-crypto/node ./cli/aes-pass-encrypt.js'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2021-06-07T11_28_10_440Z-debug.log
```

Using the `--no-bin-links` option however, succeeds:
```bash
npm install poly-crypto --no-bin-links
added 4 packages, changed 1 package, and audited 237 packages in 40s
found 0 vulnerabilities
```

Details:
```bash
node --version
v16.3.0
npm --version
7.16.0
```

To me it seems the chmod operation takes the wrong path and thus failes.

**Comment by hermannagaria** [comment]
Can you confirm on the solution above ?
I tried to do "npm install poly-crypto --no-bin-links" but still can not deploy correctly into Heroku.
The same ENOENT error occurs.

**Comment by kensnyder** [comment]
This was fixed with version 2.0.3.
